calculator.js

```diff
@@ -0,0 +1,50 @@
+function calculateMe(form) {
+    let firstDigit = form.firstNumber.value;
+    let secondDigit = form.secondNumber.value;
+    const operator = form.operator.value;
+    let result = "";
+
+    isNaN(firstDigit) ? 
+        result = result + "!!! WRONG DATA TYPE WARNING !!! \n Number must be entered on the first position.\n\n"
+        :
+        firstDigit = Number(firstDigit);
+    
+    isNaN(secondDigit) ? 
+        result = result + "!!! WRONG DATA TYPE WARNING !!! \n Number must be entered on the second position.\n\n"
+        :
+        secondDigit = Number(secondDigit);
+    
+    operator != '+' 
+        && operator != '-' 
+        && operator != '*' 
+        && operator != '/' ? 
+        result = result + "!!! OPERATOR MISMATCH WARNING !!!\n" + operator + " has a wrong data format.\n\n"
```

Comment:
how can I switch another operator if we have only 4 cases in select? Do you need this check here?

Reply:
You will be surprised how users can crack anything if they are boring... :-)
As I know, there is a rule: if you get any form from a user, at least minimal checks must be applied.
These are minimal.